ucm version 0.5.21

Okkonen-GitHub opened the pull request:
Upgrade ucm to the latest version.
I hope I did it correctly.

Okkonen-GitHub committed:
ucm version 0.5.21

2m commented:
Thank you very much! License SHA also changed, so I updated it in #17 (was not able to push to your branch)

Okkonen-GitHub commented:
Alright, great!

On Sun, 16 Jun 2024, 23.32 Martynas Mickevičius, ***@***.***>
wrote:

> Thank you very much! License SHA also changed, so I updated it in #17
> <https://github.com/2m/ucm-bin-pkgbuild/pull/17> (was not able to push to
> your branch)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/2m/ucm-bin-pkgbuild/pull/16#issuecomment-2171869355>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AON4HH2AMNGQHN5J7Y7IYOTZHXY4ZAVCNFSM6AAAAABJM2HZJ2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDCNZRHA3DSMZVGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

2m commented:
New package version has been pushed to AUR.
